Write HaploPainter files without backend installed

the-x-at opened the pull request:
Instead of implementing a new function `writeHaplopainterFile` as suggested by @jorainer, I opted to interfere as little as possible with the source code. Everything else looked like code or at least interface duplication, something I am not very happy with. There is simply an option in `switchPlotFun` that allows to skip the `HaploPainter` installation check. It still defaults to the previous behavior and will not break existing code.

- [x] `R CMD check` successful for R-4.1.0 BioC 3.13.
- [x] Updated documentation.
- [x] Updated unit tests.

Fixes issue #23.

the-x-at committed:
Fix missing character conversions in KG test

Fixes issue #2. After five years.

the-x-at committed:
Output HaploPainter input files w/o executable

Enable writing HaploPainter input files without having the
plotting backend installed. Fixes #23.

jorainer reviewed (changes requested):
Thanks for the PR @the-x-at , makes sense. 
I would however suggest to also document the option `device = "txt"` and refer there also to the example you added.

the-x-at commented on man/plotting-functions.Rd:
```diff
@@ -168,17 +176,23 @@ switchPlotfun(method)
   or png device, while, if \code{kinship2} is used, the plot can also be
   directly plotted and displayed (if \code{device="plot"} is specified).
 
   \code{HaploPainter} plotting supports also \code{device = "txt"} in
```
Option `device = "txt"` is described here. Isn't that enough?

jorainer replied:
I would also mention the option `"txt"` where the `device` parameter is described. I don't have the documentation in front of me right now but there should be somewhere an `\item{device}` or similar.

the-x-at replied:
Got it. Done with the last commit.

the-x-at committed:
Update documentation

Details on invocation of switchPlotfun to produce text files without
haplopainter installed (argument "device").

jorainer reviewed (approved):
Excellent! Thanks @the-x-at
